README.md

```diff
@@ -8,7 +8,7 @@ a portfolio resume that showcases examples of my works along with the usual resu
 
 ## Live Demo
 
-[Live Demo Link](https://github.com/binhussen)
+[Live Demo Link](https://binhussen.github.io/portfolio/)
```

Comment:
- [ ] [optional] @Team great work implementing the changes, it would be nice to ensure the live demo is the latest version of your app currently it doesn't have a contact section.

Reply:
It is because the contact section waiting for approval to be merged into the main branch.

Reply:
Thank you for your feedback, I will modify it.